Code/Framework/AzFramework/AzFramework/Physics/Material/PhysicsMaterialManager.cpp

```diff
@@ -62,24 +62,27 @@ namespace Physics
             return materialFound;
         }
 
-        // Take a reference so we can mutate it.
-        AZ::Data::Asset<AZ::Data::AssetData> assetLocal = materialAsset;
-        if (!assetLocal.IsReady())
+        auto newMaterial = CreateMaterialInternal(id, materialAsset);
+        if (newMaterial)
         {
-            assetLocal.QueueLoad();
-            assetLocal.BlockUntilLoadComplete();
-
-            // Failed to load the asset
-            if (!assetLocal.IsReady())
-            {
-                return nullptr;
-            }
+            m_materials.emplace(newMaterial->GetId(), newMaterial);
         }
 
-        auto newMaterial = CreateMaterialInternal(id, assetLocal);
-        if (newMaterial)
+        // Block until the material asset is loaded.
+        // It's important that the loading of physics material assets is not delayed,
+        // otherwise objects would react differently during simulation during a few
+        // frames until the material is loaded, causing unexpected behaviours.
```

Comment:
uber nit: We're recommended to use American English in code and comments so this should be `behavior` I'm afraid 😅

Reply:
Thanks for letting me know, I confuse them :)